src/types/types.ts

```diff
@@ -113,6 +113,7 @@ export const SURVEY_TIME_CONSTANT = '2020-06-15T00:14:04.322Z'
 export const SURVEY_IDENTIFIER = 'mindkind'
```

Comment:
One suggestion, can we move all constants into a file named: constants 
and keep types separate